packages/fxa-auth-server/lib/routes/security-events.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = (log, db, config) => {
+  return [
+    {
+      method: 'GET',
```

Comment:
Lets add the corresponding `DEL` route for security events plus tests etc.

Reply:
Sure